research/gnn-survey/README.md

```diff
@@ -88,10 +88,27 @@ python train.py \
   --sparse_features=True
 ```
 
+*   GIN results on cora (81.7% average test accuracy):
+
+`python train.py \
```

Comment:
Enclose in three backticks so it's formatted properly

Reply:
OK!